index.js

```diff
@@ -31,9 +31,9 @@ function clangFormat(file, enc, style, done) {
 function spawnClangFormat(args, done, stdio) {
   // WARNING: This function's interface should stay stable across versions for the cross-version
   // loading below to work.
-  if (args.indexOf('-version') !== -1 || args.indexOf('--version') !== -1) {
+  if (args.indexOf('--version') !== -1) {
```

Comment:
Roger, will modify to do that. I'll also squash everything into a single commit.